### ISSUE TITLE
无法复现例子中“西游记”的结果

### ISSUE THREAD

**Issue by chen2000** [nghuyong/ERNIE-Pytorch#22]
尝试用例子中西游记填空，得到的是并不是列出的结果，请问是什么原因？

结果如下：
Predicted token is: ['p450', '废', '録', '哔', '岐', '微', '谋', '衷', '谋', '奸', '微', 'fine', '嘘', '尻', '尻', '鸣', 'note', '##nny', '亏', '稗', '愚', '筌', '筌', '遛', '鬻', '嘘', '吹', '筌', '违', '戕', '戮', '鬻', '缉', '抛', '醍', '亏', 'lp', '##book', '楮', '衷', '嘘', '醪', '戮', '衷', '纂']

代码如下：
#!/usr/bin/env python
#encoding: utf-8

import torch
from pytorch_transformers import BertTokenizer, BertModel,BertForMaskedLM
 
tokenizer = BertTokenizer.from_pretrained('./ERNIE-converted/ERNIE_1.0_max-len-512-pytorch')
 
input_tx = "[CLS] [MASK] [MASK] [MASK] 是中国神魔小说的经典之作，与《三国演义》《水浒传》《红楼梦》并称为中国古典四大名著。[SEP]"
tokenized_text = tokenizer.tokenize(input_tx)
indexed_tokens = tokenizer.convert_tokens_to_ids(tokenized_text)
 
tokens_tensor = torch.tensor([indexed_tokens])
segments_tensors = torch.tensor([[0]*len(tokenized_text)])
 
model = BertForMaskedLM.from_pretrained('./ERNIE-converted/ERNIE_1.0_max-len-512-pytorch')
model.eval()

with torch.no_grad():
	outputs = model(tokens_tensor, token_type_ids=segments_tensors)
	predictions = outputs[0]

predicted_index = [torch.argmax(predictions[0, i]).item() for i in range(0,(len(tokenized_text)-1))]
predicted_token = [tokenizer.convert_ids_to_tokens([predicted_index[x]])[0] for x in range(1,(len(tokenized_text)-1))]
 
print('Predicted token is:',predicted_token)

**Comment by lxy444** [comment]
同样有此疑问，请问楼主解决了吗

**Comment by ZCcaptain** [comment]
同求

**Comment by nghuyong** [comment]
已经更新readme啦～

**Comment by RockNHawk** [comment]
参考这里 https://github.com/nghuyong/ERNIE-Pytorch/issues/68

复现这个例子是 ErnieForMaskedLM 而不是 ErnieModel
